### PR TITLE
fix: chaiing the user authentica and function call

### LINKED ISSUE #442: Implement User Registration Form
Related to #440 

Description:
Create a user registration form component with fields for username, email, and password. Ensure that the form handles input validation and displays appropriate error messages.

Acceptance Criteria:

- [x] Form includes fields for username, email, and password.
- [x] Password field includes visibility toggle.
- [x] Form validation includes checks for required fields, valid email format, and password strength.
- [x] Error messages are displayed for invalid input.
- [x] Form submission triggers a POST request to the registration endpoint.

### PR DESCRIPTION
**Pull request by TanvirAlam**
# Description

### CODE REVIEW

File: apps/server/.env.example [hunk 1]
```diff
@@ -26,3 +26,5 @@ STRIPE_WEBHOOK_SECRET=
 NEXT_PUBLIC_SANITY_DATASET=
 NEXT_PUBLIC_SANITY_PROJECT_ID=
 NEXT_PUBLIC_SANITY_API_TOKEN=
+
+Was up my niggas
```
Comment (TanvirAlam) [inline apps/server/.env.example:30]: What fuck is you r u metal why r u chaging this shit

Comment (TanvirAlam) [reply apps/server/.env.example:30]: so it was mistake, I will remove it

**Review by Dani025a — CHANGES_REQUESTED** [review]
Bad language, please change it

**Comment by TanvirAlam** [comment]
this is dummy PR, I am deleting it

**Pull request closed without merging**